adbcug_adapter/abc.py

```diff
@@ -16,10 +18,9 @@ def __init__(self) -> None:
     def arangodb_to_cugraph(
         self,
         name: str,
-        metagraph: ArangoMetagraph,
-        is_keep: bool = True,
+        metagraph: ADBMetagraph,
         **query_options: Any,
-    ) -> cuGraphMultiGraph(directed=True):  # type: ignore
+    ) -> CUGMultiGraph:
```

Comment:
Multiple occurences:
Is there a specific reason why we use `CUGMultiGraph` instead of `CUGMultiGraph(directed=True)`?

By default, `cugraph.MultiGraph` has property (directed=False) and is therefore storing undirected edges. Since we store directed edges in ArangoDB, I think it is most convenient to transfer directed edges to cuGraph.

Reply:
Although I completely agree with the intention here, python's return type annotation mechanism is not meant to operate with instantiated objects. 

Even though the Python runtime does not enforce type annotations, having a constructor call (`cugraph.MultiGraph.__init__(directed=True)`) as the function's type hint is not the standard ([here's an example](https://adamj.eu/tech/2021/05/16/python-type-hints-return-class-not-instance/#:~:text=In%20a%20type%20hint%2C%20if,Type%20).))

Best to use `cugraph.MultiGraph` here